lib/diagnostic.js

```diff
@@ -6,23 +6,49 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (inString) {
+  const words = inString.split(/\s+/)
+
+  const normalizedWords = []
+  for (let i = 0; i < words.length; i++) {
+    const word = words[i].replace(/\W+/, '')
+    normalizedWords[i] = word.toUpperCase()
+    return normalizedWords
+  }
 }
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+
+const uniqueWords = function (inString) {
+  const normalizeArray = normalizeWords(inString)
```

Comment:
Make sure `normalizeWords` is in the scope of this function, otherwise `uniqueWords` will not know about it!